spec/support/have_content_tag_for_matcher.rb

```diff
@@ -1,9 +1,24 @@
 require 'rspec/expectations'
 require 'action_view/record_identifier'
 
+module ContentTag
+  module TestExampleHelpers
```

Comment:
yup, use ContentTag::TestExampleHelpers::FeatureHelpers...

but why not just declare this directly in the matcher?